### PR TITLE
[2.19.x] G-9224 prevent search when location input is not selected

### PR DESCRIPTION
**Pull request by cassandrabailey293**
#### What does this PR do?
Prevents the user from executing an anyGeo search if location option is not selected. It also adds an inline error for this case. 
#### Who is reviewing it? 

@andrewzimmer @hayleynorton @zta6 @abel-connexta 
#### Select relevant component teams: 

@codice/ui 
#### How should this be tested?

1. build, profile install
2. make an advanced search and select anyGeo. DO NOT select a location option
3. Verify that the inline error pops up
4. Try to search and verify that your search is prevented with a toast message 
5. Go back and select a Location Option
6. Verify that the inline error goes away and you can search
#### What are the relevant tickets?
Fixes: #____

#### Screenshots

Screen Shot 2020-10-20 at 1 47 32 PM
Screen Shot 2020-10-20 at 1 47 39 PM

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Review by zta6 — APPROVED** [review]

**Review by shaundmorris — APPROVED** [review]

**Review by mojogitoverhere — APPROVED** [review]

**Comment by mojogitoverhere** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/location/location.js [hunk 1]
```diff
@@ -79,12 +80,21 @@ const Root = styled.div`
   height: ${props => (props.isOpen ? 'auto' : props.theme.minimumButtonSize)};
 `
 
+const ErrorWrapper = styled.div`
+margin-top: ${props => props.theme.minimumSpacing};
```
Comment (hayleynorton) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/location/location.js:84]: format

**Comment by cassandrabailey293** [comment]
build now

**Comment by cassandrabailey293** [comment]
build now

**Comment by cassandrabailey293** [comment]
build now

**Comment by abel-connexta** [comment]
hero in progress

**Comment by abel-connexta** [comment]
:tada: Hero Successful :tada:
Validated Search filter had inline error when no location is selected and Search is prevented when location input is not selected. Checked for all geometry search types (anyGeo, location, center) 
LGTM!

**Review by hayleynorton — APPROVED** [review]

**Comment by mojogitoverhere** [comment]
build now

**Pull request merged**